docs/releases/5.0.txt

```diff
@@ -429,6 +431,12 @@ Miscellaneous
   odd number of operands rather than exactly one operand. This is consistent
   with the behavior of MySQL, MariaDB, and Python.
 
+* The admin's ``RelatedFieldWidgetWrapper`` now subclasses ``ChoiceWidget``
```

Comment:
would this best placed under the admin specific section?

Reply:
Nope, this is under the "Backwards incompatible changes in X.Y" section which tends not to break things out by package. I don't think it's that big a change that it needs its own heading - it's likely that 99.99% of people will never know anything changed here.